student.rb

```diff
@@ -2,7 +2,9 @@
 
 # Represents a student, inherited from the Person class.
 class Student < Person
-  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+  attr_accessor :classroom
+
+  def initialize(age, classroom, name, parent_permission: true)
     super(age, parent_permission: parent_permission, name: name)
     @classroom = classroom
   end
```

Comment:
- Hey, let's establish the relationship as a student belongs to a classroom by adding the method to establish that below:


```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```